.spelling

```diff
@@ -576,6 +576,7 @@ xnames
 Yum
 zeroization
 zeroize
+Zypper
```

Comment:
do we need to include upper and lowercase?  I am not sure how the spelling works, I haven't done docs since that went in